openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/penalites.py

```diff
@@ -1,92 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
@fpagnoux : It is the penalties a company should pay if ti didn't enroll enough disabled or young people and so on.
@laem you may need it !